docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1798 @@
+{
```

Comment:
<p>Maybe I'm confused, but according to the equation above, shouldn't P(Y = 3) be equal to Phi(tau_3) - Phi(tau_2)? But then, if there's tau_3 it means there are 4 categories (because number of categories is number of thresholds + 1)</p><p> </p><p>So when you are interested in Pr(Y = k) the result is 1 - Pr(Y = k - 1). The one is because we have Phi(infty) which is 1 because Phi is a CDF.</p><p> </p><p>So I'm thinking as I'm writing, but if we want to have it very complete we should write</p><p> </p><p> </p><p>P(Y = k) = F(tau_k) - F(-infty) = F(tau_k) - 0 if k = 1</p><p>P(Y = k) = F(tau_k) - F(tau_{k-1}) if  k in {2, 3, 4, ..., K - 1}</p><p>P(Y = k) = F(+infty) - F(tau_{k-1}) if  k = K</p><p> </p><p>So if we have three categories, then the probabilities are</p><p> </p><p>P(Y = 1) = F(tau_1)</p><p>P(Y = 2) = F(tau_2) - F(tau_1)</p><p>P(Y = 3) = 1 - F(tau_2)</p><p> </p><p> </p><p>Do you think we can update this part accordingly?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Good eye. That is wrong. It should be $K-1$ thresholds where $K$ is the number of categories. Yes, changes are incorporated. :)</p>